tests/rules/test_quoted_strings.py

```diff
@@ -595,3 +595,338 @@ def test_allow_quoted_quotes(self):
                    "foo1: '[barbaz]'\n"
                    "foo2: '[bar\"baz]'\n",
                    conf)
+
+
+class QuotedKeysTestCase(RuleTestCase):
+    rule_id = 'quoted-strings'
+
+    def conf(self, options):
+        conf_with_options = (f"{self.rule_id}:\n"
+                             "  check-keys: true\n")
+        for option in options:
+            conf_with_options += f"  {option}\n"
+        return conf_with_options
```

Comment:
If I understand correctly this function allows writing:
```python
        conf = self.conf(['required: only-when-needed',
                          'extra-allowed: [^ftp://]',
                          'extra-required: [^http://]'])
```
instead:
```python
        conf = ('quoted-strings: {check-keys: true\n'
                '                 required: only-when-needed\n'
                '                 extra-allowed: [^ftp://]\n'
                '                 extra-required: [^http://]\n')
```

If you don't mind I prefer the classic version, because:
\- there isn't a big added value in number of lines
\- in my opinion it's a bit worse for readability
\- the rest of yamllint code uses the classic version.

What do you think?

Reply:
Sure.